server/CGameHandler.cpp

```diff
@@ -1436,10 +1341,14 @@ int CGameHandler::moveStack(int stack, BattleHex dest)
 	ret = path.second;
 
 	int creSpeed = gs->curB->tacticDistance ? GameConstants::BFIELD_SIZE : curStack->Speed(0, true);
+	bool hasLongBridge = vstd::contains_if(battleGetAllObstaclesOnPos(BattleHex(ESiegeHex::GATE_BRIDGE), false), [](const std::shared_ptr<const CObstacleInstance> obst)
```

Comment:
Perhaps rename to "hasWideMoat"? 
Same goes to other places with same variable name

Reply:
done